sycl/source/detail/global_handler.cpp

```diff
@@ -354,6 +359,12 @@ void shutdown_late() {
   if (!Handler)
     return;
 
+#if defined(XPTI_ENABLE_INSTRUMENTATION) && defined(_WIN32)
+  if (xptiTraceEnabled())
+    return; // When doing xpti tracing, we can't safely shutdown on Win.
+            // TODO: figure out why XPTI prevents release.
```

Comment:
Just for good measure, make sure we have a tracker (internal or external) for this.

Reply:
CMPLRLLVM-67135